shed/event_streams.py

```diff
@@ -254,33 +257,31 @@ def curate_streams(self, nds, outbound):
         If we get multiple streams make (name, (doc, doc, doc, ...))
         Otherwise (name, (doc,))
         """
+        # if there are multiple streams
+        if isinstance(nds[0], tuple):
```

Comment:
from tests, inputs and expected outputs still occur as expected?
One question, is this input acceptable? `(('start', {}). )`
It's not tested. On the run but I thought I'd quickly ask this.
```python
In [20]: es.curate_streams((('foo',2 ), ), True)
Out[20]: ('foo', 2)

In [21]: es.curate_streams((('foo',2 ), ), False)
Out[21]: ('foo', (2,))
```
I'd say output is as I'd expect.